content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+those identities.
+
+{{< image width="50%"
+    link="./multi-cluster.svg"
+    alt="Multiple service meshes with certificate authorities"
+    title="Trust between meshes"
+    caption="Multiple service meshes with certificate authorities"
+    >}}
+
+## Tenancy models
+
+In Istio, a **tenant** is a group of users that share
+common access and privileges to a set of deployed workloads. Generally, you
+isolate the workload instances from multiple tenants from each other through
+network configuration and policies.
+
+You can configure tenancy models to satisfy the following organizational
+requirements for isolation:
+
+- Security
+- Policy
+- Capacity
+- Cost
+- Performance
+
+Istio supports two types of tenancy models:
+
+- [Namespace tenancy](#namespace-tenancy)
+- [Cluster tenancy](#cluster-tenancy)
+
+### Namespace tenancy
+
+In Istio, a [namespace](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-namespace)
+is a unit of tenancy within a single mesh. For example, you can grant a team
+permission to deploy their workloads only to a given namespace or set of
+namespaces. By default, services from multiple tenant namespaces can communicate
+with each other.
+
+{{< image width="50%"
+    link="./iso-ns.svg"
+    alt="A service mesh with two isolated namespaces"
+    title="Isolated namespaces"
+    caption="A service mesh with two isolated namespaces"
+    >}}
+
+To improve isolation, you can selectively choose which services to expose to
+other namespaces. You can configure authorization policies for exposed services
+to restrict access to only the appropriate callers.
+
+{{< image width="50%"
+    link="./exp-ns.svg"
+    alt="A service mesh with two namespaces and an exposed service"
+    title="Namespaces with an exposed service"
+    caption="A service mesh with two namespaces and an exposed service"
+    >}}
+
+When using [multiple clusters](#multiple-clusters), the
+namespaces in each cluster sharing the same name are considered as the
+same namespace. For example, `foo/B` in `cluster-1` and `foo/B` in
+`cluster-2` refer to the same service, and Istio merges their endpoints for
+service discovery and load balancing.
+
+{{< image width="50%"
+    link="./cluster-ns.svg"
+    alt="A service mesh with two clusters with the same namespace"
+    title="Multi-cluster namespaces"
+    caption="A service mesh with clusters with the same namespace"
+    >}}
+
+### Cluster tenancy
+
+Istio supports using clusters as a unit of tenancy. In this case, you can give
+each team a dedicated cluster or set of clusters to deploy their
+workloads. Permissions for a cluster are usually limited to the members of the
+team that owns it. You can set various roles for finer grained control, for
+example:
+
+- Cluster administrator
+- Developer
+
+With cluster tenancy, Istio considers each cluster as an independent mesh.
+
+{{< image width="50%"
+    link="./cluster-iso.svg"
```

Comment:
Should have Business Unit 1 and Business Unit 2 (missing the 2).

Reply:
Yes, I'll add the missing no.